ci/travis/macos/before_install.sh

```diff
@@ -13,29 +13,51 @@
 #                                                                         #
 ###########################################################################
 
-# Remove default gdal provided by travis  (we will replace it with gdal 2)
-brew remove gdal || true
+echo `date +%s` > /tmp/travis_timestamp
 
 brew tap osgeo/osgeo4mac
 brew update
-brew install osgeo/osgeo4mac/qgis-214 --without-postgresql --only-dependencies
-brew install spawn-fcgi
-brew install lighttpd
-brew install poppler
-brew install bison
-brew install expat
+brew install python3
 
-brew ln bison --force
-brew ln sqlite --force
-brew ln openssl --force
-brew ln expat --force
-brew ln libxml2 --force
-brew ln gettext --force
-brew ln libffi --force
+pip3 install \
+    numpy \
+    psycopg2 \
+    numpy \
+    nose2 \
+    pyyaml \
+    mock \
+    future \
+    termcolor
 
-mkdir -p ${HOME}/Library/Python/2.7/lib/python/site-packages
-echo 'import site; site.addsitedir("/usr/local/lib/python2.7/site-packages")' >> ${HOME}/Library/Python/2.7/lib/python/site-packages/homebrew.pth
-echo 'import site; site.addsitedir("/usr/local/opt/gdal-20/lib/python2.7/site-packages")' >> ${HOME}/Library/Python/2.7/lib/python/site-packages/gdal2.pth
+brew install \
+    qscintilla2 \
+    qt \
+    qt5-webkit \
+    qca \
+    qwtpolar \
+    gsl \
+    sqlite \
+    expat \
+    proj \
+    gdal2-python --with-python3 \
+    spawn-fcgi \
+    lighttpd \
+    poppler \
+    bison \
+    expat \
+    bison \
+    flex \
+    ninja \
+    ccache \
+    spatialindex \
+    fastcgi \
+    qtkeychain \
+    gnu-sed
 
-# Needed for Processing
-pip install psycopg2 numpy nose2 pyyaml mock future
+brew link bison --force
+brew link flex --force
```

Comment:
Not sure forcing the install of these are still necessary if using the CMAKE_PREFIX_PATH prefixes in `install.sh`